vllm_ascend/models/__init__.py

```diff
@@ -41,6 +41,10 @@ def register_model():
             "DeepseekV3ForCausalLM",
             "vllm_ascend.models.deepseek_dbo:CustomDeepseekDBOForCausalLM")
 
+        ModelRegistry.register_model(
+            "Qwen3MoeForCausalLM",
+            "vllm_ascend.models.qwen3_dbo:CustomQwen3MoeForCausalLMDBO")
```

Comment:
I suggested to keep same name rule with deepseek one CustomQwen3MoeDBOForCausalLM